tools/check-changelogger-use.php

```diff
@@ -128,15 +173,31 @@ function debug() {
 $exit = 0;
 foreach ( $touched_projects as $slug => $files ) {
 	if ( empty( $ok_projects[ $slug ] ) ) {
-		printf( "---\n" ); // Bracket message containing newlines for better visibility in GH's logs.
-		printf(
-			"::error::Project %s is being changed, but no change file in %s is touched!%%0A%%0AGo to that project and use `%s add` to add a change file.%%0AGuidelines: https://github.com/Automattic/jetpack/blob/master/docs/writing-a-good-changelog-entry.md\n",
-			$slug,
-			"projects/$slug/{$changelogger_projects[ $slug ]['changes-dir']}/",
-			'packages/changelogger' === $slug ? 'bin/changelogger' : 'vendor/bin/changelogger'
-		);
-		printf( "---\n" );
-		$exit = 1;
+		if ( $list ) {
+			echo "$slug\n";
+		} elseif ( getenv( 'CI' ) ) {
+			printf( "---\n" ); // Bracket message containing newlines for better visibility in GH's logs.
+			printf(
+				"::error::Project %s is being changed, but no change file in %s is touched!%%0A%%0AUse `jetpack changelogger add %s` to add a change file.%%0AGuidelines: https://github.com/Automattic/jetpack/blob/master/docs/writing-a-good-changelog-entry.md\n",
+				$slug,
+				"projects/$slug/{$changelogger_projects[ $slug ]['changes-dir']}/",
+				$slug
+			);
+			printf( "---\n" );
+			$exit = 1;
+		} else {
+			printf(
+				"\e[1;31mProject %s is being changed, but no change file in %s is touched!\e[0m\n",
```

Comment:
Would saying `was added` make more sense?

Reply:
The code intentionally allows modifications and deletions to satisfy the check.

Deletions in particular are counted so as to not require special logic to try to detect a release PR like #20214. If deletions didn't count (and we didn't figure out some other way to detect a release PR), the modifications to `projects/plugins/jetpack/composer.json`, `projects/plugins/jetpack/jetpack.php`, `projects/plugins/jetpack/package.json`, and `projects/plugins/jetpack/readme.txt` in that PR would have required a change entry, which in turn would have required unwanted "-alpha" version bumps in several of those files.

Reply:
Ah gotcha, thanks for clarifying.